1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,14 @@
 function readingList(books) {
-  // Write your code here...
+  const container = document.getElementById('content');
+  let booksHTML = '';
+  books.forEach((book) => {
+    const bgColor = book.alreadyRead ? 'green' : 'red';
+    booksHTML = booksHTML.concat(`<li class="list-group-item d-flex flex-column align-items-center mb-1" style="background-color:${bgColor}">
+    <p class="text-md-center font-weight-bold text-light">${book.title} by ${book.author}</p>
+    <img style="max-width: 300px" src="${book.bookCoverImage}" />    
+    </li>`);
```

Comment:
Using string literals and string concatenation is not recommended when composing DOM elements, as this practice makes the code brittle, inflexible, hard to read, and prone to security attacks.

You should use the standard DOM manipulation API for this task (e.g. document.createElement, classList.add, append, appendChild, etc.).